CMakeLists.txt

```diff
@@ -21,6 +21,8 @@ set(RGL_AUTO_TAPE_PATH "" CACHE STRING  # STRING prevents from expanding relativ
 # Test configuration
 set(RGL_BUILD_TESTS ON CACHE BOOL
     "Enables building test. GTest will be automatically downloaded")
+set(RGL_BUILD_TAPED_TESTS OFF CACHE BOOL
+    "Enables building taped test.")
```

Comment:
GTest should also be downloaded for `RGL_BUILD_TAPED_TESTS`. Please take a look at [external/CMakeLists.txt](https://github.com/RobotecAI/RobotecGPULidar/blob/develop/external/CMakeLists.txt#L53)